.circleci/config.yml

```diff
@@ -1,6 +1,6 @@
 version: 2.1
 orbs:
-  cypress: cypress-io/cypress@1
+  cypress: cypress-io/cypress@1.27.0
```

Comment:
weird, i thought you already changed this in the last PR...

Reply:
Oh, this was a branch off that branch so this got left behind, haha. I'll take it back out.